quic/s2n-quic-transport/src/space/rx_packet_numbers/ack_ranges.rs

```diff
@@ -25,14 +30,21 @@ impl AckRanges {
     }
 
     /// Inserts a packet number; dropping smaller values if needed
-    pub fn insert_packet_number(&mut self, packet_number: PacketNumber) -> bool {
-        if self.0.insert_value(packet_number).is_ok() {
+    #[inline]
+    pub fn insert_packet_number_range(&mut self, pn_range: PacketNumberRange) -> bool {
+        let interval = (
+            Bound::Included(pn_range.start()),
+            Bound::Included(pn_range.end()),
+        );
+        if self.0.insert(interval).is_ok() {
             return true;
         } else {
```

Comment:
if you're returning early you don't need the `else` so just dedent the whole thing